src/librustdoc/templates/page.html

```diff
@@ -29,77 +29,79 @@
         >
     {%- endfor -%}
     <script id="default-settings" {# -#}
-      {% for k, v in layout.default_settings %}
+      {% for (k, v) in layout.default_settings %}
         data-{{k}}="{{v}}"
       {%- endfor -%}
     ></script> {#- -#}
-    <script src="{{static_root_path | safe}}storage{{page.resource_suffix}}.js"></script> {#- -#}
-    <script src="{{page.root_path | safe}}crates{{page.resource_suffix}}.js"></script> {#- -#}
-    <script defer src="{{static_root_path | safe}}main{{page.resource_suffix}}.js"></script> {#- -#}
+    <script src="{{static_root_path|safe}}storage{{page.resource_suffix}}.js"></script> {#- -#}
+    <script src="{{page.root_path|safe}}crates{{page.resource_suffix}}.js"></script> {#- -#}
+    <script defer src="{{static_root_path|safe}}main{{page.resource_suffix}}.js"></script> {#- -#}
     {%- for script in page.static_extra_scripts -%}
-    <script defer src="{{static_root_path | safe}}{{script}}.js"></script> {#- -#}
+    <script defer src="{{static_root_path|safe}}{{script}}.js"></script> {#- -#}
     {% endfor %}
     {%- if layout.scrape_examples_extension -%}
-    <script defer src="{{page.root_path | safe}}scrape-examples{{page.resource_suffix}}.js"></script> {#- -#}
+    <script defer src="{{page.root_path|safe}}scrape-examples{{page.resource_suffix}}.js"></script> {#- -#}
     {%- endif -%}
     {%- for script in page.extra_scripts -%}
-    <script defer src="{{page.root_path | safe}}{{script}}.js"></script> {#- -#}
+    <script defer src="{{page.root_path|safe}}{{script}}.js"></script> {#- -#}
     {% endfor %}
     <noscript> {#- -#}
         <link rel="stylesheet" {# -#}
-           href="{{static_root_path | safe}}noscript{{page.resource_suffix}}.css"> {#- -#}
+           href="{{static_root_path|safe}}noscript{{page.resource_suffix}}.css"> {#- -#}
     </noscript> {#- -#}
-    {%- if layout.css_file_extension -%}
-        <link rel="stylesheet" type="text/css" {# -#}
-            href="{{static_root_path | safe}}theme{{page.resource_suffix}}.css"> {#- -#}
-    {%- endif -%}
-    {%- if layout.favicon -%}
+    {%- match layout.css_file_extension -%}
+        {%- when Some with (ext) -%}
```

Comment:
`ext` winds up unused here. Is there a way to express that in the match? Or do, e.g. `if layout.css_file_extension.is_some()`?

Reply:
Yup, that works. Queued it up in my local working copy.